vtex-apps/admin-pages
# Fix: StoreForm typing

#### What is the purpose of this pull request?

Improve `StoreForm`'s typing and refactor some parts of it.

#### What problem is this solving?

Poor typing.

#### How should this be manually tested?
1. Use the Store Editor;
2. Check if it still works. :)

#### Screenshots or example usage
Workspace: [https://fixstoreformtypings--storecomponents.myvtex.com/admin/cms/storefront/](https://fixstoreformtypings--storecomponents.myvtex.com/admin/cms/storefront/).
